app/models/EpicTests.scala

```diff
@@ -60,6 +60,7 @@ case class ArticlesViewedSettings(
 
 case class EpicTest(
   name: String,
+  nickname: String = "",
```

Comment:
when adding a new field it's preferable to make it optional:
`nickname: Option[String]`